pkgs/by-name/fr/freewheeling/package.nix

```diff
@@ -54,20 +47,11 @@ stdenv.mkDerivation (finalAttrs: {
     libsndfile
     libogg
     libSM
-    (gnutls.overrideAttrs (oldAttrs: {
-      configureFlags = oldAttrs.configureFlags ++ [ "--enable-openssl-compatibility" ];
-    }))
+    libX11
+    nettle
   ];
-  env.NIX_CFLAGS_COMPILE = toString (
-    makeSDLFlags [
-      SDL
-      SDL_ttf
-      SDL_gfx
```

Comment:
Were these possibly needed in a previous version and just never removed? Same with some other dependencies you appear to have been able to just remove..

Reply:
These dependencies still exist, but are now being located by `pkg-config`.
Afaict, only gnutls was actually unused.